e2e/tests/regression/test_v2.robot

```diff
@@ -116,3 +116,88 @@ V2 Volume Should Cleanup Resources When Instance Manager Is Deleted
         And Check volume 1 data is intact
         And Check volume 2 data is intact
     END
+
+Test V2 Data Engine Selective Activation
+    # create volumes with 2 replicas on node 0 and node 1
+    # there is no replica on node 2
+    Given Create volume 0 attached to node 0 with 2 replicas excluding node 2    dataEngine=v2
+    And Create volume 1 attached to node 0 with 2 replicas excluding node 2    dataEngine=v1
+
+    When Label node 2 with node.longhorn.io/disable-v2-data-engine=true
+    Then Check v2 instance manager is not running on node 2
+    And Check v1 instance manager is running on node 2
+
+    When Label node 1 with node.longhorn.io/disable-v2-data-engine=true
+    Then Check v2 instance manager is running on node 1
+    And Check v1 instance manager is running on node 1
+
+    When Detach volume 0
+    And Wait for volume 0 detached
+    Then Check v2 instance manager is not running on node 2
+    And Check v1 instance manager is running on node 2
```

Comment:
According to https://github.com/longhorn/longhorn/issues/7015#issuecomment-1808553645 step 6, should test script also check v2 instance manager not running node 1 as well?

Reply:
Yes, you're right! Thanks for pointing this out!